### PR TITLE
Finish main function of image processing using openCV

### PR DESCRIPTION
**Pull request by DominikKwiatkowski**

### CODE REVIEW

File: drones/image_processing/config.ini [hunk 1]
```diff
@@ -1,3 +1,8 @@
 [COLOR_RANGE]
 LOWER_BOUND = 0 220 100
 UPPER_BOUND = 15 255 255
+
+[OBJECT]
+WIDTH = 7.8
+FOCAL = 1800
+FOV = 82.6
```
Comment (melehean) [inline drones/image_processing/config.ini]: What is FOV? It should be precise imo

File: drones/image_processing/utils.py [hunk 2]
```diff
@@ -6,6 +6,39 @@
 import drones.image_processing.normalization as normalization
 
 
+def process_image(image: np.ndarray) -> typing.Tuple[float, float, float]:
+    """
+    Detect object, counter it distance from camera,pitch and yaw
```
Comment (Qubaef) [inline drones/image_processing/utils.py]: ?

File: drones/image_processing/utils.py [hunk 3]
```diff
@@ -6,6 +6,39 @@
...
+    Parameters:
+    ----------
+    image: np.ndarray
+        image to be processed
+
+    Returns:
+    ----------
+        direction and distance: typing.Tuple[float,float,float]
+        first two values are direction to object(yaw and pitch). They are countered from image position. Distance is
+        countered from real width of object and focal length of camera.
```
Comment (Qubaef) [inline drones/image_processing/utils.py]: ?

File: drones/image_processing/utils.py [hunk 4]
```diff
@@ -6,6 +6,39 @@
...
+    config_parser.read("image_processing/config.ini")
+    config = config_parser["OBJECT"]
+
+    focal = int(config["FOCAL"])
+    real_width = float(config["WIDTH"])
+    ((x, y), width) = detect_object(image)
+    if x > 0:
+        distance = distance_to_camera(real_width, focal, width)
+        vector = vector_to_centre(image.shape[1], image.shape[0], (x, y), 0.5)
+        return (
```
Comment (melehean) [inline drones/image_processing/utils.py:37]: This return is quite hard to read. Maybe more precise variable naming

**Review by Qubaef — CHANGES_REQUESTED** [review]
Fix typos in comments.

**Review by Qubaef — APPROVED** [review]

**Pull request merged**